### PR TITLE
Add link to no stores error

### LINKED ISSUE #553: Rename: Error: You must have created at least one store
The message: 
> Error: You must have created at least one store 

is clear, but does not provide instructions on where are stores located. 

My suggestion is to replace with:

> Error: You must create a store (with a **link** to /stores/create) 

Suggested message not only gives a warring but also gives clear action on what to do next and eliminates UX confusion.

### PR DESCRIPTION
**Pull request by Kukks**
Closes #553

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Argh no, by doing this, I need to review all code calling StatusMessage to know if it is weak to injection.
Better a Temp data property, and use it on the view.

File: BTCPayServer/Controllers/AppsController.cs [hunk 1]
```diff
@@ -73,7 +73,12 @@ public async Task<IActionResult> CreateApp()
             var stores = await _AppsHelper.GetOwnedStores(GetUserId());
             if (stores.Length == 0)
             {
-                StatusMessage = "Error: You must have created at least one store";
+                StatusMessage = new StatusMessageModel()
+                {
+                    Html =
+                        $"Error: You must have created at least one store. <a href='{(Url.Action("CreateStore", "UserStores"))}'>Create store</a>",
+                    Severity = "danger"
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/AppsController.cs]: while you are at it, you can use an enum? :D

Comment (Kukks) [reply BTCPayServer/Controllers/AppsController.cs]: bah!

**Comment by Kukks** [comment]
ready @NicolasDorier  but I know you wanted to rape the code more :P

**Comment by NicolasDorier** [comment]
For now, I would prefer that you make a simple test to know if the first character is a `{` to know if it is json before the try/catch. The reason is that raising exceptions are expensive.

**Comment by Kukks** [comment]
good to go @NicolasDorier

**Comment by britttttk** [comment]
tACK :)

**Pull request merged**